eywa/math/numpy_backend.py

```diff
@@ -1,8 +1,10 @@
+# flake8: noqa 
```

Comment:
@farizrahman4u This one line in the file breaks the tests for numba. The rest all works properly. Tested. :+1: Should I remove this? It runs properly then.

Reply:
I dont get it, what has backend got anything to do with this?
They way it should work is, there should be 3 jobs: one for numba backend, one for numpy and one for pep8.

The pep8 job shouldn't run any tests, just check pep8.

Reply:
You can check the keras repo for reference.

Reply:
@farizrahman4u I'm not quite sure how flake8 can be run on it's own without running tests. I'll look into it.

Reply:
Any updates? Thanks.

Reply:
https://github.com/keras-team/keras/blob/adfec1e8440b3af64b642ea75bdb21bbee70cc80/.travis.yml#L10 This the PEP8 tests on keras, right?

Reply:
Yes, and here.. https://github.com/keras-team/keras/blob/adfec1e8440b3af64b642ea75bdb21bbee70cc80/.travis.yml#L110